test/optimus/pr/test_optimus_PR.wdl

```diff
@@ -40,11 +39,9 @@ workflow TestOptimusPR {
       bam = target.bam,
       matrix = target.matrix,
       matrix_summary = target.matrix_summary,
-      picard_metrics = target.picard_metrics,
       expected_bam_hash = expected_bam_hash,
       expected_matrix_hash = expected_matrix_hash,
-      expected_matrix_summary_hash = expected_matrix_summary_hash,
-      expected_picard_metrics_hash = expected_picard_metrics_hash
+      expected_matrix_summary_hash = expected_matrix_summary_hash
```

Comment:
Is there a way to test expected gene and cell metrics like this test wdl was previously doing with `picard_metrics`?

Reply:
I believe that what we're testing is deterministic, it should be possible. 

I excluded it because I think there's going to be a lot of churn on how the metrics get calculated, but that's probably not a good reason. I'll add the structure back in to make the tests and verify determinism.